### ISSUE TITLE
Error in syncing Discord to Cabal

### ISSUE THREAD

**Issue by ninabreznik** [timgoeller/cabal-discord-bridge#12]
Just noticed messages don't get synced to Cabal anymore. Maybe new Cabal updates? If I restart glitch server, it syncs pending but then dooesn't sync the newcoming again :(

![Screenshot_2021-07-29_18-40-53](https://user-images.githubusercontent.com/1076427/127539836-5ed19007-8a4a-44d9-beb2-8ad493a98b05.png)

https://glitch.com/edit/#!/dat-ecosystem-bot?path=README.md%3A1%3A0

**Comment by timgoeller** [comment]
Yeah, sounds like a Cabal update. I'll take a look at it later! 👍

**Comment by ninabreznik** [comment]
Thanks, Tim. @Alexander Cobleigh ***@***.***> mentioned it could also
be that it doesn't sync because I am locally running a different version
that the bot. Not sure how to fix this though to work on any version 🤷‍♀️
